### ISSUE TITLE
Using -q twice reverts it

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#449]
Originally reported of Google Code with ID 393.

```
In htools running with -qq will actually remove the quiet option.
The manpage says:

       -q, --quiet
              Decrease the output verbosity.  Each usage of this option will decrease the verbosity (less than zero doesn't make  sense)
              from the default of one.

So basically this contraddicts the manpage, although it's just a cosmetic issue.
```
Originally added on 2013-03-02 04:18:01 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I am a bit surprised to hear this, as the code is:

oQuiet :: OptType
oQuiet =
  (Option "q" ["quiet"]
   (NoArg (\ opts -> Ok opts { optVerbose = optVerbose opts - 1 }))
   "decrease the verbosity level",
   OptComplNone)

So there's no toggling or anything like that. Can you show exactly how this doesn't work? And in what tool?

Ooh, I see. There were two checks in hbal where it was tested for equality to zero, and these were copied to the other tools (hinfo, hcheck). Will fix.
```
Originally added on 2013-03-04 10:23:13 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit bfa99f7
Author: Iustin Pop <iustin@google.com>
Date:   Mon Mar 4 11:24:51 2013 +0100

    Fix low verbosity levels in htools
    
    In a few cases, we tested the verbosity level for (== 0), instead of
    higher/lower than a certain value. If the user passes multiple
    "--quiet" options, this can result in negative verbosity levels, which
    behave like "extra verbosity".
    
    Sigh for plain data types as opposed to proper verbosity levels…
    
    Signed-off-by: Iustin Pop <iustin@google.com>
    Reviewed-by: Guido Trotter <ultrotter@google.com>

```
Originally added on 2013-03-04 10:41:13 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
